lib/interface/collection/artist.dart

```diff
@@ -252,7 +255,8 @@ class ArtistTile extends StatelessWidget {
                         palette = result.colors;
                       }
                       await precacheImage(getAlbumArt(artist), context);
-                      MobileNowPlayingController.instance.hide();
+                      if (!Configuration.instance.stickyMiniplayer)
```

Comment:
I'm still confused as to why this condition could not be placed directly inside `MobileNowPlayingController.instance.hide` instead of checking it everywhere in the UI.

Reply:
i didn't want to edit a function but if it's the proper way then im leaving it to you, + there might be some situations where you want to force hide the miniplayer (which won't work if u placed it inside the `hide()`)